XamlControlsGallery/ControlPages/GridViewPage.xaml

```diff
@@ -14,7 +14,9 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:local="using:AppUIBasics"
-    xmlns:local1="using:AppUIBasics.ControlPages">
+    xmlns:local1="using:AppUIBasics.ControlPages"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)">
```

Comment:
I don't think you're using this anywhere, are you?

Reply:
Yes, removed that line. Thanks!